shell/platform/darwin/ios/framework/Source/SemanticsObject.mm

```diff
@@ -610,14 +610,13 @@ - (BOOL)onCustomAccessibilityAction:(FlutterCustomAccessibilityAction*)action {
 - (NSString*)accessibilityLabel {
   if (![self isAccessibilityBridgeAlive])
     return nil;
-  NSString* label;
+  NSString* label = nil;
   if (![self node].label.empty()) {
     label = @([self node].label.data());
   }
   if (![self node].tooltip.empty()) {
-    label = [NSString stringWithFormat:@"%@%@",
-                                       label ? [NSString stringWithFormat:@"%@\n", label] : @"",
-                                       @([self node].tooltip.data())];
+    label = label ? [NSString stringWithFormat:@"%@\n%@", label, @([self node].tooltip.data())]
```

Comment:
same logic but cleaner

Reply:
cc @jmagman for readability

Reply:
You could also use a `NSMutableString` to build up the string.  This works too, though.